src/test/java/DropdownTest.java

```diff
@@ -0,0 +1,20 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DropdownTest extends BasicSetupTest{
+    @Test
+    public void dropdownOption2Test() throws InterruptedException {
+
+        browser.get("https://the-internet.herokuapp.com/dropdown");
+
+        WebElement dropdown = browser.findElement(By.xpath("//*[@id=\"dropdown\"]"));
+        dropdown.click();
+        Thread.sleep(2000);
```

Comment:
Despite we weren't aware at that time that `Thread.sleep(X)` is overall a bad practice - here it's a bit an overkill as the state change after selecting a dropdown happens immediately (as it doesn't require any loading)

So this one and one below can be removed without any loss - if kept, they can highly impact the test execution duration, as you can imagine in the framework you can have thousands of tests like this which means if at least one `Thread.sleep(2000)` per test, it will cost 1000 * 2s = 2000s ~ 33 minutes of time lost for no reason

Reply:
A bit more to read: https://www.selenium.dev/documentation/webdriver/waits/